### PR TITLE
PMM-8050 Release Notes 2.17.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

**Comment by PaulJacobs-percona** [comment]
https://6094d40396688d00071debb4--pmm-doc.netlify.app/release-notes/2.17.0.html

File: docs/release-notes/2.17.0.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Single-line install of PMM Server on supported Linux distros (this feature is in Technical Preview).
+
+- DBaaS Changes: (this feature is in Technical Preview)
+    - DBaaS is now is better open for the public, so you can quickly turn it ON  on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
+    - Database components management will enable PMM administrators to restrict some DB versions from creating DB Clusters.
+    - For PXC clusters created using DBaaS, the HAProxy will be used by default.  Please note: the proper monitoring HAProxy with DBaaS will come later.
+
+- Changes to *Sign in to Percona Platform*. From this release, Registration of the Percona account will be more secure and require additional confirmation.
+
+(???) **  Collecting data about Security Threat tool and  Integrated Alerting usage. To better understand features adoption and force our resources to needed features, we are adding and additional Telemetry data to be collected. The data will be boolean values is the feature enabled not on the PMM Server. Please note: all data is anonymous and no is PII collected.
```
Comment (PaulJacobs-percona) [inline docs/release-notes/2.17.0.md:40]: @shoffman-percona Needed?

File: docs/release-notes/2.17.0.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## New Features
+
+* [PMM-7863](https://jira.percona.com/browse/PMM-7863): [BE] Add version management for HAProxy
+* [PMM-7848](https://jira.percona.com/browse/PMM-7848), [PMM-7847](https://jira.percona.com/browse/PMM-7847), [PMM-7421](https://jira.percona.com/browse/PMM-7421): Add support for using SSL certificates between pmm-admin and monitored MySQL databases
+* [PMM-7883](https://jira.percona.com/browse/PMM-7883): Single-line install of PMM Server on supported Linux distros - [Technical Preview]
+* [PMM-7013](https://jira.percona.com/browse/PMM-7013), [PMM-7819](https://jira.percona.com/browse/PMM-7819): [BE] Use HAProxy by default instead of ProxySQL
+* [PMM-7356](https://jira.percona.com/browse/PMM-7356), [PMM-7581](https://jira.percona.com/browse/PMM-7581): [FE] Management of available versions of components
+* [PMM-7358](https://jira.percona.com/browse/PMM-7358), [PMM-7576](https://jira.percona.com/browse/PMM-7576): [FE] Default version for Create DB cluster
+* [PMM-7518](https://jira.percona.com/browse/PMM-7518): [FE] Backup: Simple restore - SAME Service [MySQL]
```
Comment (rnovikovP) [inline docs/release-notes/2.17.0.md:51]:
```suggestion
```

Removing the line. Skipped for this release, the full feature is not ready

File: docs/release-notes/2.17.0.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    - For PXC clusters created using DBaaS, the HAProxy will be used by default.  Please note: the proper monitoring HAProxy with DBaaS will come later.
+
+- Changes to *Sign in to Percona Platform*. From this release, Registration of the Percona account will be more secure and require additional confirmation.
+
+(???) **  Collecting data about Security Threat tool and  Integrated Alerting usage. To better understand features adoption and force our resources to needed features, we are adding and additional Telemetry data to be collected. The data will be boolean values is the feature enabled not on the PMM Server. Please note: all data is anonymous and no is PII collected.
+
+
+## New Features
+
+* [PMM-7863](https://jira.percona.com/browse/PMM-7863): [BE] Add version management for HAProxy
```
Comment (rnovikovP) [inline docs/release-notes/2.17.0.md:45]: @denisok is this makes sense now without FE part? AFAIK user can't use this for now

File: docs/release-notes/2.17.0.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* [PMM-7013](https://jira.percona.com/browse/PMM-7013), [PMM-7819](https://jira.percona.com/browse/PMM-7819): [BE] Use HAProxy by default instead of ProxySQL
+* [PMM-7356](https://jira.percona.com/browse/PMM-7356), [PMM-7581](https://jira.percona.com/browse/PMM-7581): [FE] Management of available versions of components
+* [PMM-7358](https://jira.percona.com/browse/PMM-7358), [PMM-7576](https://jira.percona.com/browse/PMM-7576): [FE] Default version for Create DB cluster
+* [PMM-7518](https://jira.percona.com/browse/PMM-7518): [FE] Backup: Simple restore - SAME Service [MySQL]
+
+## Improvements
+
+* [PMM-7572](https://jira.percona.com/browse/PMM-7572): Add TLS options to mysqld_exporter
+* [PMM-7783](https://jira.percona.com/browse/PMM-7783): Support of pg_stat_monitor v0.9.0
+* [PMM-7064](https://jira.percona.com/browse/PMM-7064): Alert Rules List - Severity Colors
```
Comment (rnovikovP) [inline docs/release-notes/2.17.0.md:57]:
```suggestion
* [PMM-7064](https://jira.percona.com/browse/PMM-7064): Integrated Alerting: Presenting severity of the Alert Rule using different colors
```

File: docs/release-notes/2.17.0.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* [PMM-7884](https://jira.percona.com/browse/PMM-7884): DBaaS: Fix DB Cluster tab loading
+* [PMM-7941](https://jira.percona.com/browse/PMM-7941): mongodb_exporter provides a wrong replication status value
+* [PMM-7917](https://jira.percona.com/browse/PMM-7917): Postgres exporter has high CPU usage during Restart
+* [PMM-8037](https://jira.percona.com/browse/PMM-8037): Platform Sign in flow on PMM broken
+* [PMM-7702](https://jira.percona.com/browse/PMM-7702): DBaaS: Cannot edit PSMDB clusters
+* [PMM-7991](https://jira.percona.com/browse/PMM-7991): MySQL Summary panel doesn't exist on MySQL Summary dashboard
+* [PMM-7939](https://jira.percona.com/browse/PMM-7939): Inconsistent reported version
+* [PMM-7920](https://jira.percona.com/browse/PMM-7920): Postgres Exporter has increased memory usage with pmm-client 2.15.1 & pmm-server 2.16.0
+* [PMM-7700](https://jira.percona.com/browse/PMM-7700): Rule API crashing - undefined params
+* [PMM-7616](https://jira.percona.com/browse/PMM-7616): IA: Wrong HTML title
```
Comment (rnovikovP) [inline docs/release-notes/2.17.0.md:74]:
```suggestion
* [PMM-7616](https://jira.percona.com/browse/PMM-7616): Integrated Alerting: Incorrect title of the page in a browser 
```

File: docs/release-notes/2.17.0.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+- Single-line install of PMM Server on supported Linux distros (this feature is in Technical Preview).
+
+- DBaaS Changes: (this feature is in Technical Preview)
+    - DBaaS is now is better open for the public, so you can quickly turn it ON  on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
+    - Database components management will enable PMM administrators to restrict some DB versions from creating DB Clusters.
+    - For PXC clusters created using DBaaS, the HAProxy will be used by default.  Please note: the proper monitoring HAProxy with DBaaS will come later.
+
+- Changes to *Sign in to Percona Platform*. From this release, Registration of the Percona account will be more secure and require additional confirmation.
+
+(???) **  Collecting data about Security Threat tool and  Integrated Alerting usage. To better understand features adoption and force our resources to needed features, we are adding and additional Telemetry data to be collected. The data will be boolean values is the feature enabled not on the PMM Server. Please note: all data is anonymous and no is PII collected.
```
Comment (rnovikovP) [inline docs/release-notes/2.17.0.md:40]:
```suggestion
```

remove this from the highlight. adding to the Improvements

File: docs/release-notes/2.17.0.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+## Improvements
+
+* [PMM-7572](https://jira.percona.com/browse/PMM-7572): Add TLS options to mysqld_exporter
+* [PMM-7783](https://jira.percona.com/browse/PMM-7783): Support of pg_stat_monitor v0.9.0
+* [PMM-7064](https://jira.percona.com/browse/PMM-7064): Integrated Alerting: Presenting severity of the Alert Rule using different colors
+* [PMM-7946](https://jira.percona.com/browse/PMM-7946): Better error message on PMM client if server doesn't support HAProxy
+* [PMM-7932](https://jira.percona.com/browse/PMM-7932): Usability improvements on UI for adding External Services
+* [PMM-7641](https://jira.percona.com/browse/PMM-7641), [PMM-7820](https://jira.percona.com/browse/PMM-7820): [BE] Make DBAAS feature flag - same as other Feature Flags
+* [PMM-7966](https://jira.percona.com/browse/PMM-7966): Collect basic Telemetry data about Integrated Alerting and Security Threat Tool usage
```
Comment (rnovikovP) [inline docs/release-notes/2.17.0.md:59]:
```suggestion
* [PMM-7966](https://jira.percona.com/browse/PMM-7966): Telemetry: Collect enabled/disabled status for Integrated Alerting and Security Threat Tool features
```

File: docs/release-notes/2.17.0.md (hunk 3 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.17.0.md]: 7863 & 7013 have different explanations: default vs version management. If same, which is it?

**Pull request merged**